### DISCUSSION TITLE
Is there a way to install in LXC base on Alpine

### DISCUSSION THREAD

**Discussion by seb49** [Koenkk/zigbee2mqtt#13643] [General]
Hello, 

Is there a way to install zigbee2mqtt in a LXC Proxmox container base on alpine ?

I try many things but I not good enough in linux and docker world (I try to reproduce step in docker file in my Alpine install)

Regards

**Comment by Nick4-1** [comment]
A little late but in case someone else bumps on this: if you take a look at tteck's proxmox helper scripts on https://tteck.github.io/Proxmox there is a script there to do this in a jiffy for you.
